nixos/modules/installer/sd-card/sd-image-aarch64.nix

```diff
@@ -39,6 +39,17 @@
         # Supported in newer board revisions
         arm_boost=1
 
+        [cm4]
+        kernel=u-boot-rpi4.bin
+        enable_gic=1
+        armstub=armstub8-gic.bin
+        disable_overscan=1
```

Comment:
I'm feeling a bit lazy, and always have trouble finding the section on the vendor's website.

Wouldn't leaving those four options out end-up using the values in the `[pi4]` section? If it's needed, no problem. (You'll need the `[cm4]` section for `otg_mode=1`...)

Reply:
It's needed because we need to set `otg_mode`.

Reply:
But the other options in the cm4 section, which are duplicated from the rpi4 section. Those shouldn't be needed, right?

Reply:
I'm not confident. Let me test.

Reply:
Indeed, that works just fine.

Pushed a commit removing the duplicated bits.